snmp.yml

```diff
@@ -1565,3 +1565,132 @@ fortinet_fortigate:
           type: Integer32
         - labelname: fgWcWtpSessionWtpId
           type: OctetString
+
+# ServerTech Sentry 3 MIB
+#
+# Used by ServerTech PDUs
+#
+# ftp://ftp.servertech.com/Pub/SNMP/sentry3/Sentry3OIDTree.txt
+# ftp://ftp.servertech.com/Pub/SNMP/sentry3/Sentry3.mib
+servertech_sentry3:
+  version: 2
+  auth:
+    community: public
+  walk:
+    - 1.3.6.1.2.1.1
+    - 1.3.6.1.4.1.1718.3.2.2.1
+    - 1.3.6.1.4.1.1718.3.2.3.1
+  metrics:
+    - name: sysUpTime
+      oid: 1.3.6.1.2.1.1.3
+    # Infeed metrics
+    - name: infeed_load_hundredths_amper
```

Comment:
If you want to stick to something that could be machine generated, and fit with Prometheus naming convention, how about something like this:

`infeed_load_value_hundredth_amps` or `infeedLoadValue_hundredth_amps`